components/formats-bsd/src/loci/formats/in/OMETiffReader.java

```diff
@@ -1056,7 +1063,14 @@ public MetadataStore getMetadataStoreForConversion() {
     int realSeries = getSeries();
     for (int i=0; i<getSeriesCount(); i++) {
       setSeries(i);
-      store.setPixelsBinDataBigEndian(new Boolean(!isLittleEndian()), i, 0);
+      if (meta.getPixelsBinDataCount(i) > 0) {
+        for (int j=0; j<meta.getPixelsBinDataCount(i); j++) {
+          store.setPixelsBinDataBigEndian(new Boolean(!isLittleEndian()), i, j);
+        }
+      }
+      else {
+        store.setPixelsBigEndian(Boolean.TRUE, i);
```

Comment:
```
git grep -l getPixelsBinDataBigEndian
components/bio-formats-plugins/src/loci/plugins/out/Exporter.java
components/formats-api/src/loci/formats/MetadataTools.java
components/formats-api/src/loci/formats/meta/MetadataConverter.java
components/formats-bsd/src/loci/formats/gui/AWTImageTools.java
components/formats-bsd/src/loci/formats/gui/BufferedImageWriter.java
components/formats-bsd/src/loci/formats/in/OMEXMLReader.java
components/formats-bsd/src/loci/formats/out/APNGWriter.java
components/formats-bsd/src/loci/formats/out/ICSWriter.java
components/formats-bsd/src/loci/formats/out/JPEG2000Writer.java
components/formats-bsd/src/loci/formats/out/JavaWriter.java
components/formats-bsd/src/loci/formats/out/OMEXMLWriter.java
components/formats-bsd/src/loci/formats/out/TiffWriter.java
components/formats-bsd/src/loci/formats/out/V3DrawWriter.java
components/test-suite/src/loci/tests/testng/FormatReaderTest.java
cpp/lib/ome/xml/meta/Convert.cpp
```

Basically every writer.  But the logic is generally (from `TiffWriter`):

```
    if (retrieve.getPixelsBigEndian(series) != null) {
      littleEndian = !retrieve.getPixelsBigEndian(series).booleanValue();
    }
    else if (retrieve.getPixelsBinDataCount(series) == 0) {
      littleEndian = !retrieve.getPixelsBinDataBigEndian(series, 0).booleanValue();
    }
```

So long as every writer is using `getPixelsBigEndian` and only using `getPixelsBinDataBigEndian` as a fallback, then it looks like it's safe to drop this from `OmeTiffReader`.  Arguably, they should only be using `getPixelsBigEndian`, since it's the reader's responsibility to set this correctly.  The fallback shouldn't be needed in every writer.